packages/pages/src/vite-plugin/plugin.ts

```diff
@@ -5,28 +5,14 @@ import {
   ProjectStructureConfig,
 } from "../common/src/project/structure.js";
 import { build } from "./build/build.js";
-import merge from "lodash/merge.js";
 
-/**
- * Options to configure functionality of the plugin.
- *
- * @public
- */
-export type Options = {
-  projectStructureConfig?: Optional<ProjectStructureConfig>;
```

Comment:
just double checking, none of the options in this config have any effect if the user passes it into the plugin?

Reply:
I see, so it's unlikely that this change would break something for a user? if so, should there be another item made to look into this further if these are configurations that are expected to be available and work?